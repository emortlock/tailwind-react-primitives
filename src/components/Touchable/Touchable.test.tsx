import React from 'react'
import { render, RenderResult } from '@testing-library/react'

import { Touchable } from '.'

const setup = (
  testProps = {},
): [RenderResult, { props: Record<string, any> }] => {
  const props = Object.assign({}, testProps)

  const wrapper = render(<Touchable {...props}>Hello World</Touchable>)

  return [
    wrapper,
    {
      props,
    },
  ]
}

describe('Touchable', () => {
  it('renders matching snapshot', () => {
    const [{ asFragment }] = setup()

    expect(asFragment()).toMatchSnapshot()
  })
})
