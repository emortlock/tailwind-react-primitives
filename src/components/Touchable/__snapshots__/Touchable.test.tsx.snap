// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Touchable renders matching snapshot 1`] = `
<DocumentFragment>
  <button
    class="select-none cursor-pointer focus:outline-none focus:shadow-outline"
  >
    Hello World
  </button>
</DocumentFragment>
`;
